.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     working_directory: /code
     docker:
         - image: openjdk:10
-        - image: jboss/keycloak:latest
+        - image: jboss/keycloak:7.0.1
```

Comment:
There is already 8

Reply:
Yeah, that is why I have had to force it to test on 7.
We are going to do this release for 7.0.1, and then work on a new release that accommodates the changes in keycloak 8 as they will not be backwards compatible